util/syn_yosys.sh

```diff
@@ -10,59 +10,139 @@
 # The following tools are required:
 #  - sv2v: SystemVerilog-to-Verilog converter from github.com/zachjs/sv2v
 #  - yosys: synthesis tool from github.com/YosysHQ/yosys
+#  - Cadence Conformal
 #
 # Usage:
-#   syn_yosys.sh 2>&1 | tee syn.std
+#   ./syn_yosys.sh |& tee syn.std
 
 #-------------------------------------------------------------------------
 # use fusesoc to generate files and file list
 #-------------------------------------------------------------------------
-\rm -Rf build
-fusesoc --cores-root .. run --target=sim --setup --build formal > /dev/null 2>&1
+rm -Rf build syn_out
+
+fusesoc --cores-root .. run --tool=icarus --target=syn \
+  --setup "lowrisc:systems:top_earlgrey" > /dev/null 2>&1
 
 # copy all files into directory "syn_out"
-\rm -Rf syn_out
 mkdir syn_out
-cp build/formal_0/src/*/*/*.sv build/formal_0/src/*/*/*/*.sv syn_out
+cp build/*/src/*/*.sv* build/*/src/*/*/*.sv* build/*/src/*/*/*/*.sv* syn_out
+cd syn_out
+
+# copy file list and remove incdir lines
+grep -v incdir ../build/*/*/*.scr > flist_gold
+
+# remove all hierarchical paths
+sed -i 's!.*/!!' flist_gold
```

Comment:
You could combine these two by piping the output of `grep` into `basename`. It might be a bit clearer what's going on? (If you don't want to do that, I don't think "hierarchical paths" is really the right name. Just "Remove directories from paths" maybe?)

Reply:
Great suggestion, I'll try it out in the new PR